test-clients/src/main/java/com/hedera/services/bdd/suites/token/UniqueTokenBurnWipeSpecs.java

```diff
@@ -0,0 +1,101 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.ou may not use this file except in compliance with the License.
+ *  * You may obtain a copy of the License at
+ *  *
+ *  *     http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ *  * ‍
+ *
+ *
+ */
+
+package com.hedera.services.bdd.suites.token;
+
+import com.google.protobuf.ByteString;
+import com.hedera.services.bdd.spec.HapiApiSpec;
+import com.hedera.services.bdd.suites.HapiApiSuite;
+import com.hederahashgraph.api.proto.java.TokenSupplyType;
+import com.hederahashgraph.api.proto.java.TokenType;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.List;
+
+import static com.hedera.services.bdd.spec.queries.QueryVerbs.getReceipt;
+import static com.hedera.services.bdd.spec.queries.QueryVerbs.getTxnRecord;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.burnToken;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.cryptoCreate;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.mintToken;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.tokenCreate;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.newKeyNamed;
+
+public class UniqueTokenBurnWipeSpecs extends HapiApiSuite {
+	private static final org.apache.logging.log4j.Logger log = LogManager.getLogger(UniqueTokenBurnWipeSpecs.class);
+	private static final String TOKEN_NAME = "token";
+	private static final String SUPPLY_KEY = "supplyKey";
+	private static final String TREASURY = "treasury";
+
+	public static void main(String... args) {
+		new UniqueTokenBurnWipeSpecs().runSuiteSync();
+	}
+
+	@Override
+	protected List<HapiApiSpec> getSpecsInSuite() {
+		return List.of(new HapiApiSpec[]{
+				burnHappyPath()
+				}
+		);
+	}
+
+	private HapiApiSpec burnHappyPath(){
+		return HapiApiSpec.defaultHapiSpec("burnHappyPath")
+				.given(
+						newKeyNamed(SUPPLY_KEY),
+						cryptoCreate(TOKEN_TREASURY),
+						tokenCreate(TOKEN_NAME)
+								.tokenType(TokenType.NON_FUNGIBLE_UNIQUE)
+								.supplyType(TokenSupplyType.INFINITE)
+								.initialSupply(0)
+								.supplyKey(SUPPLY_KEY)
+								.treasury(TREASURY),
+
+						mintToken(TOKEN_NAME, 10),
+						mintToken(TOKEN_NAME, List.of(ByteString.copyFromUtf8("memo")))
+				).when(
+						burnToken(TOKEN_NAME, List.of(1L)).via("burnTxn")
+				).then(
+						getTxnRecord("burnTxn").logged(),
+						getReceipt("burnTxn").logged()
+				);
+	}
+
+	@Override
+	protected Logger getResultsLogger() {
+		return log;
+	}
+
+
```

Comment:
This is temporal, I had some merge conflicts and decided to extract it in a separate file for now. Will refactor it to be in the original unique e2e spec